src/commands/lm/info.js

```diff
@@ -0,0 +1,40 @@
+const Listr = require('listr')
+
+const Command = require('../../utils/command')
+const {
+  checkGitVersionStep,
+  checkGitLFSVersionStep,
+  checkLFSFiltersStep,
+  checkHelperVersionStep,
+} = require('../../utils/lm/steps')
+
+class LmInfoCommand extends Command {
+  async run() {
+    const steps = [
+      checkGitVersionStep,
+      checkGitLFSVersionStep,
+      checkLFSFiltersStep((ctx, task, installed) => {
+        if (!installed) {
+          throw new Error('Git LFS filters are not installed, run `git lfs install` to install them')
+        }
+      }),
+      checkHelperVersionStep,
+    ]
+
+    await this.config.runHook('analytics', {
+      eventName: 'command',
+      payload: {
+        command: 'lm:info',
+      },
+    })
+
+    const tasks = new Listr(steps, { concurrent: true, exitOnError: false })
+    tasks.run().catch(() => {})
```

Comment:
Should the error be printed instead of ignored?

Reply:
Great point!
That suppress a redundant error (which is already printed when a task fails):
![image](https://user-images.githubusercontent.com/26760571/107260271-a1382b80-6a46-11eb-8e8a-9ecc19254b73.png)